miniserver/routes/index.js

```diff
@@ -0,0 +1,141 @@
+const express = require('express')
+const config = require('../config')
+const router = express.Router()
+const { StatusCodes } = require('http-status-codes')
+const { Logger } = require('../logger')
+const NodeCache = require('node-cache')
+const Cache = new NodeCache()
+const { phone } = require('phone')
+const Twilio = require('twilio')
+const utils = require('../utils')
+const { User } = require('../src/data/user')
+const { isEqual, pick } = require('lodash')
+const stringify = require('json-stable-stringify')
+const { Setting } = require('../src/data/setting')
+const { Request } = require('../src/data/request')
+const blockchain = require('../blockchain')
+const { ethers } = require('ethers')
+
+/**
+* @openapi
+* /health:
+*   get:
+*     description: Checks the health of the miniwallet-server
+*     responses:
+*       200:
+*         description: OK indicates server is running
+*/
+router.get('/health', async (req, res) => {
+  Logger.log('[/health]', req.fingerprint)
+  res.send('OK').end()
+})
+
+const checkfromTwilio = async (req, res, next) => {
+  // check valid Twilio Request
+  if (!Twilio.validateRequest(config.twilio.token, req.headers['x-twilio-signature'], ('https://' + req.get('host') + '/sms'), req.body)) {
```

Comment:
use req.protocol and req.path

Reply:
Done : Note have ignored the twilio checking when in debug mode as ngrok maps https to http and we may want to test using curl or postman for development.